app/models/user.rb

```diff
@@ -1,3 +1,6 @@
+# rubocop:disable Lint/Void
+
+# rubocop:disable Lint/ShadowingOuterLocalVariable
 class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
[required] It’s better to delete commented piece of codes. If you need to review a previous version of your code, use git history